### PR TITLE
Configurable global notification banner

### LINKED ISSUE #2363: Hide global notification banner by default
### Problem description
New MAGDA builds currently display a global notification banner at the top of every page with the following message:

> A new look for Australia's data portal: our updated site makes it easier for you to find relevant open data. You can still **go back to the old site (link)**

While the banner component is still relevant for application-specific notifications, it should probably be hidden by default. Also, the default message displayed should be generic and not include links to legacy websites.

### Problem reproduction steps

1. Navigate to the homepage of a new MAGDA instance
2. See banner

### Note

- It might be good to make this banner optional and configurable.

### PR DESCRIPTION
**Pull request by sajidanower23**
### What this PR does

Fixes #2363 

The global notification banner should be configurable now through the `showNotificationBanner` flag (defaults to `false`).

### Checklist

-   [x] ~There are unit tests to verify my changes are correct~ Unit tests aren't applicable
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (core dev team only)

### CODE REVIEW

File: deploy/helm/magda/charts/web-server/templates/configmap.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ data:
     {{- if .Values.fallbackUrl -}} "fallbackUrl": {{ .Values.fallbackUrl | quote }}, {{- end -}}
     {{- if .Values.contentApiBaseUrlInternal -}} "contentApiBaseUrlInternal": {{ .Values.contentApiBaseUrlInternal | quote }}, {{- end -}}
     {{- if .Values.defaultOrganizationId -}} "defaultOrganizationId": {{ .Values.defaultOrganizationId | quote }}, {{- end -}}
+    "showNotificationBanner": {{ .Values.showNotificationBanner }}
```
Comment (AlexGilleran) [inline deploy/helm/magda/charts/web-server/templates/configmap.yaml]: You're missing a `,` here which is breaking the config map

File: deploy/helm/minikube-esri-dev.yml [hunk 2]
```diff
@@ -31,13 +31,14 @@ registry-api:
...
-  
+
 correspondence-api:
   smtpPort: 2525
   smtpHostname: "smtp.mailgun.org"
-  
+
 web-server:
   fallbackUrl: "https://data.gov.au"
+  showNotificationBanner: false
```
Comment (AlexGilleran) [inline deploy/helm/minikube-esri-dev.yml]: If you set it to `false` in magda-web/values.yml it'll default to false and you won't have to set it in all of these deployment specific yamls :)

Comment (sajidanower23) [reply deploy/helm/minikube-esri-dev.yml]: Done

**Comment by sajidanower23** [comment]
@AlexGilleran made the changes you requested, also deployed successfully on to dev (deleted the deployment afterwards though).

**Pull request merged**